src/jdk.crypto.cryptoki/share/classes/sun/security/pkcs11/P11Key.java

```diff
@@ -381,6 +383,18 @@ static SecretKey pbeKey(Session session, long keyID, String algorithm,
             new CK_ATTRIBUTE(CKA_SENSITIVE),
             new CK_ATTRIBUTE(CKA_EXTRACTABLE),
         });
+        if ((SunPKCS11.mysunpkcs11 != null) && attrs[0].getBoolean() || attrs[1].getBoolean() || (attrs[2].getBoolean() == false)) {
```

Comment:
This line should be folded, and parentheses are missing here: I think around the `||` portion:
```java
        if ((SunPKCS11.mysunpkcs11 != null)
            && (attrs[0].getBoolean()
                || attrs[1].getBoolean()
                || (attrs[2].getBoolean() == false))
        ) {
```
See also lines 358-361.

Reply:
Updated